settings.ini

```diff
@@ -29,10 +29,10 @@ user = Nixtla
 requirements = numpy packaging pandas>=1.1.1
 plotting_requirements = pandas[plot] plotly plotly-resampler
 polars_requirements = polars[numpy]
-dev_requirements = black datasetsforecast==0.0.8 nbdev<2.3.26 numba>=0.58.0 pyarrow scipy
+dev_requirements = black datasetsforecast==0.0.8 nbdev<2.3.26 numba>=0.58.0 pyarrow scipy mypy flake8
```

Comment:
I've told you several times that these are sorted alphabetically. also we don't need those, we just need pre-commit

Reply:
updated to include

* missing libs from `environment.yml` and removed `environment.yml` since it was redundant.
* pre-commit which was missing originally